homeassistant/components/switch/flux.py

```diff
@@ -46,7 +47,8 @@
     vol.Optional(CONF_STOP_CT, default=1900):
         vol.All(vol.Coerce(int), vol.Range(min=1000, max=40000)),
     vol.Optional(CONF_BRIGHTNESS):
-        vol.All(vol.Coerce(int), vol.Range(min=0, max=255))
+        vol.All(vol.Coerce(int), vol.Range(min=0, max=255)),
+    vol.Optional(CONF_MODE, default="xy"): cv.string
```

Comment:
Utilize voluptuous to make sure the modes are valid:

```
MODE_XY='xy'
MODE_MIRED='mired'

DEFAULT_MODE=MODE_XY
```

`vol.Optional(CONF_MODE, default=DEFAULT_MODE): vol.Any(MODE_XY, MODE_MIRED)`